src/DwitTech.AccountService.Core/Services/ActivationService.cs

```diff
@@ -0,0 +1,61 @@
+﻿using DwitTech.AccountService.Core.Interfaces;
+using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Mail;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DwitTech.AccountService.Core.Services
+{
+    public class ActivationService : IActivationService
+    {
+        private string GetActivationCodeFromDb(string userEmail)
+        {
+            return "pttigkgkgkgkgk6666k6k6k6k6k6k6k1123ndncj";
+        }
+
+        private string GetNameFromDb(string userEmail)
```

Comment:
Whats this for?

Reply:
I put it there to retrieve the user's name for the email template.